crates/viewer/re_viewer_context/src/time_control.rs

```diff
@@ -539,34 +636,50 @@ impl TimeControl {
         self.set_time(time);
     }
 
+    pub fn get_time_for_timeline(&self, timeline: Timeline) -> Option<TimeReal> {
```

Comment:
https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

```suggestion
    pub fn time_for_timeline(&self, timeline: Timeline) -> Option<TimeReal> {
```